android/src/main/java/cl/json/social/WhatsAppShare.java

```diff
@@ -19,6 +20,24 @@ public WhatsAppShare(ReactApplicationContext reactContext) {
     @Override
     public void open(ReadableMap options) throws ActivityNotFoundException {
         super.open(options);
+        
+        if (options.hasKey("whatsAppNumber")) {
+            // create an empty conversation in case it's not on contacts
+            this.getIntent().setComponent(new ComponentName(PACKAGE, "com.whatsapp.Conversation")); 
+            this.openIntentChooser();
+
+
+            // leave room for the conversation to be created
+            try {
+                Thread.sleep(300);   
```

Comment:
We can apply the same idea about the `constant` here. Do you think it would be a good idea to share this number between the two modules?